index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,300;1,400;1,700&display=swap" rel="stylesheet">
+    <title>CLAY</title>
+</head>
+<body>
+    <header class="navigation">
+        <nav class="nav-bar">
+            <div class="nav-list">
+                <ul>
+                    <li>
+                        <img src="img/github1.png" alt="">
+                    </li>
+                    <li>
+                        <img src="img/github1.png" alt="">
+                    </li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+        </nav>
+        <nav class="navbar-2">
+            <div class="idea idea-2">
+                <div class="logo-img2">
+                    <img src="img/logo4.png" alt="">
+                </div>
+                <div class="logo-text">
+                    <h3>CLAY</h3>
+                    <h3>Ghana 2022</h3>
+                </div>
+            </div>
+            <div class="nav-list-2">
+                <ul>
+                    <li class="first-list"><a href="about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponser</li>
+                    <li>News</li>
+                    <li class="last-list">CLAY Campaign</li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <header class="welcome-PG">
+        <nav>
+            <div class="menupopup"  id="menupopup">
+                <img id="closer"  src="./img/Icon.png" alt="icon-png">
+                <ul>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Sponser</a>
+                    <a href="">News</a>
+                    <a href="">CLAY Campaign</a>
+                </ul>
+            </div>
+            <div class="humburger" id="MENU">
+               <img src="img/menubtn.png" alt="">
+            </div>
+        </nav>
+        <div class="head-contain">
+            <div class="welcome-msg">
+                <p>"Warm welcome, Heros"</p>
+                <h1>WORLD BEST ETHICAL <br> HACKERS SUMMIT 2022</h1>
+                
+            </div>
+        </div>
+    </header>
```

Comment:
**OPTIONAL** - As I suggested above, for semantic reasons, it is not advisable to have two `header` tags in one HTML page, please fix this 🙏

Reply:
I will work on that as soon as possible sir, thank you for the suggestions.